solutions/과자_나눠주기.js

```diff
@@ -0,0 +1,41 @@
+const INPUT_PATH = "./inputs/과자_나눠주기.txt";
+
+const fs = require("fs");
+const filePath = process.platform === "linux" ? "/dev/stdin" : INPUT_PATH;
+const input = fs.readFileSync(filePath).toString().trim().split("\n");
+
+function solution(inputArguments) {
+  const [N, list] = inputArguments;
+
+  const [children, _] = N.split(" ").map(Number);
+  let cookieLangthList = list.split(" ").map(Number).sort((a, b) => b - a);
```

Comment:
```suggestion
  const cookieLengthList = list.split(" ").map(Number).sort((a, b) => b - a);
```
재할당 되는 곳이 없기 때문에 const로 쓰는게 나아보입니다.